modules/ROOT/pages/create-custom-dashboard.adoc

```diff
@@ -0,0 +1,30 @@
+= Create a Custom Dashboard
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Create a custom dashboard to configure the metrics and data points you want to view.  
+
+To create a custom dashboard:
+
+. Navigate to *Custom dashboards* in Anypoint Monitoring.
```

Comment:
Should we always say "In Anypoint Platform, click *Anypoint Monitoring* in the first task of a topic?